src/test/java/org/spdx/utility/license/LicenseExpressionParserTestV2.java

```diff
@@ -226,8 +226,13 @@ public void testExternalLicenseRef() throws InvalidSPDXAnalysisException {
 		assertEquals(externalExtractedId, ((ExternalExtractedLicenseInfo)result).toString());
 	}
 
-    public void regressionMitWith() throws InvalidSPDXAnalysisException, InvalidLicenseStringException {
+    public void regressionMitWith() throws InvalidSPDXAnalysisException {
         AnyLicenseInfo result = LicenseInfoFactory.parseSPDXLicenseStringCompatV2("MIT WITH Autoconf-exception-2.0");
         assertEquals("MIT WITH Autoconf-exception-2.0",result.toString());
     }
+
+	public void testParseNoAssertionAnds() throws InvalidSPDXAnalysisException {
+		AnyLicenseInfo result = LicenseInfoFactory.parseSPDXLicenseStringCompatV2("MIT AND NOASSERTION AND NONE");
+		assertEquals("(MIT AND NOASSERTION AND NONE)",result.toString());
```

Comment:
@pmonks This part of the code targets SPDX 2.x. So it has to be separately resolved.